### PR TITLE
add json schema to core

### PR DESCRIPTION
**Pull request by MarkSackerberg**

### CODE REVIEW

File: src/pages/core/json-schema.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
+---
+title: JSON Schema
+metaTitle: JSON Schema | Core
+description: Understand the structure and requirements of the off-chain JSON schema for Metaplex Core assets.
+---
+
+The off-chain JSON metadata for Metaplex Core assets is similar to the Metaplex Token Metadata standard. However, since more data can be stored on-chain in the asset itself using plugins, some of the data, like creators, are no longer part of the JSON standard. Although they are not required, they can still be used, and some wallets and tools might render them.
```
Comment (L0STE) [inline src/pages/core/json-schema.md:7]:
```suggestion
The off-chain JSON metadata for Metaplex Core assets resembles the Metaplex Token Metadata one. However, with plugins allowing more data to be stored on-chain, fields like creators are now optional but may still be included, as some wallets and tools may render them.
```

File: src/pages/core/json-schema.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+## JSON Schema Fields
+
+Now, we will explore the various fields in the Schema. Examples for different types of Assets such as Videos, Images, 3D Files, and HTML Assets can be found at the bottom of the page. It's important to note that all of these different types can also be part of a single Asset using the `image`, `animation_url`, and `properties` fields.
+
+If you miss some fields that you knew from Metaplex Token Metadata those are probably deprecated. The `creators` for example are part of the [Royalties Plugin](core/plugins/royalties) now.
+
+### Required Fields
```
Comment (L0STE) [inline src/pages/core/json-schema.md:15]: Do you think wrapping this section up in a table like in [this section](https://developer-hub-git-core-schma-metaplex-foundation.vercel.app/core/guides/javascript/how-to-create-a-core-nft-asset-with-javascript#uploading-the-metadata) might make it a little more clean? (For both Required and Optional field)

File: src/pages/core/json-schema.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+title: JSON Schema
+metaTitle: JSON Schema | Core
+description: Understand the structure and requirements of the off-chain JSON schema for Metaplex Core assets.
+---
+
+The off-chain JSON metadata for Metaplex Core assets is similar to the Metaplex Token Metadata standard. However, since more data can be stored on-chain in the asset itself using plugins, some of the data, like creators, are no longer part of the JSON standard. Although they are not required, they can still be used, and some wallets and tools might render them.
+
+## JSON Schema Fields
+
+Now, we will explore the various fields in the Schema. Examples for different types of Assets such as Videos, Images, 3D Files, and HTML Assets can be found at the bottom of the page. It's important to note that all of these different types can also be part of a single Asset using the `image`, `animation_url`, and `properties` fields.
```
Comment (L0STE) [inline src/pages/core/json-schema.md:11]: I think we can put this text where the example are!

**Review by L0STE — CHANGES_REQUESTED** [review]

**Review by tonyboylehub — COMMENTED** [review]
Can we possibly add a copyable blank template towards the start maybe? I feel like that might be good to have here.

**Comment by MarkSackerberg** [comment]
> Can we possibly add a copyable blank template towards the start maybe? I feel like that might be good to have here.

I've changed the order to have the examples at the top.

Also addressed the other nitpicks. Feel free to have a last look @tonyboylehub @L0STE

**Review by L0STE — APPROVED** [review]

**Pull request merged**